### PR TITLE
Added a footer to the main page

### LINKED ISSUE #335: Add a footer to the main page
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

## The Problem:
We need to add a footer to the demo and provide some generic information in it, exact content of the footer is yet to be decided.

## How to Solve?
We need to add a footer to the main index.html file at the following location
https://github.com/publiclab/image-sequencer/blob/88364decf6a6111a0be31fd98010576d5c6f918e/examples/index.html#L88
It should look something like this
```js
<footer class="page-footer font-small">
 </footer>
```
Instance Count 1

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #335

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers

**Comment by Divy123** [comment]
Can you please provide a screenshot for this?
Thanks

**Comment by Harshithpabbati** [comment]
screenshot from 2019-02-15 12-43-47
@Divy123

**Comment by Harshithpabbati** [comment]
@jywarren please review it.

File: examples/index.html [hunk 1]
```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
 
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
```
Comment (Divy123) [inline examples/index.html]: I suggest that the footer section should be at the  bottom of the page, I mean there should be no space below it as well.
Can you please do this after a review by @jywarren ?
Thanks

File: examples/index.html [hunk 2]
```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
...
+          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues">Ask a question &raquo;</a>
+        </p>
+      </div>
+      <div class="col-md-6">
+        <h2>Improve this tool</h2>
+        <p>
+          This is an open source toolkit which you can help add to and improve on <a href="https://github.com/">Github</a>
+        </p>
+        <p>
+          <a class="btn btn-default" href="https://github.com/publiclab/image-sequencer">View the code</a>
```
Comment (Divy123) [inline examples/index.html]: Also sine the `ask a question` is blue in colour can you please make `view the code` also in blue?

Comment (Harshithpabbati) [reply examples/index.html]: Sure i will change it

Comment (Harshithpabbati) [reply examples/index.html]: What abt the link in ask a question do i keep it same (github issues link)??

Comment (Divy123) [reply examples/index.html]: Yes !

**Comment by Harshithpabbati** [comment]
screen
@Divy123 is this ok??

**Comment by Divy123** [comment]
Ya this one is good.

> I suggest that the footer section should be at the bottom of the page, I mean there should be no space below it as well.
Can you please do this after a review by @jywarren ?
Thanks

Lets wait for this

**Comment by Harshithpabbati** [comment]
I will send the pull request after he his review

**Comment by Harshithpabbati** [comment]
@jywarren review it :)

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

**Comment by Harshithpabbati** [comment]
@jywarren please review it.

**Comment by HarshKhandeparkar** [comment]
Harshithpabbati you will have to wait for another 24hrs before jywarren reviews this as it will be a sunday beginning for his timezone. Thanks!

**Comment by Harshithpabbati** [comment]
@jywarren please review it

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

**Comment by jywarren** [comment]
Thanks for your patience and sorry for the delay, this looks great!

**Comment by Harshithpabbati** [comment]
Thanks @jywarren

**Pull request merged**